File: tests/expected/import_tests.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

int test() {
  List<int> a = [1, 2, 3];
  return (a[1] ?? (throw Exception("key not found")));
}

main(List<String> argv) {
  final int b = test();
  assert(b == 2);
  print(sprintf("%s", ["OK"]));
}

File: tests/expected/go/import_tests.go
package main

import (
	"fmt"
)

func Test() int {
	var a []int = []int{1, 2, 3}
	return a[1]
}

func main() {
	var b int = Test()
	if !(b == 2) {
		panic("assert")
	}
	fmt.Printf("%v\n", "OK")
}
